hbase-client/src/main/java/org/apache/hadoop/hbase/replication/ReplicationPeerConfig.java

```diff
@@ -117,6 +121,10 @@ public Map<TableName, List<String>> getExcludeTableCFsMap() {
     return (Map<TableName, List<String>>) excludeTableCFsMap;
   }
 
```

Comment:
I'm not sure what the best practice here is, but would it make sense to add a deprecated setter to `ReplicationPeerConfig`?

Reply:
ReplicationPeerConfig is designed to be immutable. So you'd better add a setter in the ReplicationPeerConfigBuilder.

Reply:
I added this method to the builder in the [initial commit](https://github.com/apache/hbase/pull/5819/files#diff-4111cec0f23deab9e2827446de52e105d46d5f91e8f66b9a0c60e495b236d1e6R250). I asked this question because I noticed ReplicationPeerConfig has some deprecated setter methods. I wasn't sure if it would make sense to introduce a deprecated setter for this new field to follow the existing code, or if I should leave it out for this new field.